styles/variables-silicon.less

```diff
@@ -200,4 +188,4 @@
 // ---------------------------------------
 @agate-togglebutton-indicator-position: auto 33px 9px 33px;
 @agate-togglebutton-indicator-height: 4px;
-@agate-togglebutton-indicator-border-width: 1px;
+@agate-togglebutton-indicator-border-width: 1px;
```

Comment:
Could you add one more line? at the end of the file?

Reply:
Yes, done.